.pre-commit-hooks.yaml

```
@@ -49,6 +49,13 @@
     language: r
     files: '\.[rR](md)?$'
     minimum_pre_commit_version: "2.13.0"
+-   id: parsable-roxygen
+    name: parsable-roxygen
+    description: check if roxygen comments in a .R file are parsable
+    entry: Rscript /inst/hooks/exported/parsable-roxygen.R
```

```suggestion
    entry: Rscript inst/hooks/exported/parsable-roxygen.R
```
No trailing slash here since hooks must be relative to the package root